repos/system_upgrade/el7toel8/actors/authselectscanner/tests/unit_test.py

```diff
@@ -0,0 +1,624 @@
+import textwrap
```

Comment:
Wrong import order. Wondering why our tests didn't catch it

Reply:
Missing rebase?